src/interpret.jl

```diff
@@ -433,6 +433,20 @@ function check_isdefined(frame, @nospecialize(node))
     end
 end
 
+function coverage_visit_line!(frame::Frame)
+    pc, code = frame.pc, frame.framecode
+    code.report_coverage || return
+    src = code.src
+    codeloc = src.codelocs[pc]
```

Comment:
Can use https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/22d1914cb4fcd08de6f27a8cfe97a5f5205cfe06/src/utils.jl#L360?

Reply:
Seems like that would be less efficient, since we don't need to actually look up `codeloc` in the linetable unless the location has changed. Since this function is probably quite hot, it's probably worth doing this way.

Reply:
👍